### ISSUE TITLE
Implement UDS Communication Specification

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#35]
## User story
1. As a developer
2. I need a base implementation of the UDS spec
3. So that I can create UDS messages for the CAN bus

## Acceptance criteria
* Create messages according to docs (from #21)
* Show example sequences (e.g. in terminal)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
